docs/keyboardShortcuts.md

```diff
@@ -62,8 +62,9 @@ These shortcuts can be used when the focus is on an issue card in a panel.
 
 ## Control the panel view window ...
 
-| **Action on the browser view**| **Shortcut**|**Explanation**|
+| **Action on the panel view**| **Shortcut**|**Explanation**|
 | ------------- |:-------------:| -----------|
+| Bring HubTurbo into focus|<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>H</kbd>| Global Hotkey to make HubTurbo topmost window|
```

Comment:
I see, go ahead if it fails gracefully in other platforms. Can do as part of this or as a separate issue.